composer.json

```diff
@@ -3,7 +3,16 @@
     "description": "Simple library to talk to metrics collector services.",
     "keywords": ["metrics", "logging"],
     "authors": [
-        {"name": "Benjamin Eberlei", "email": "kontakt@beberlei.de"}
+        {
+            "name": "Benjamin Eberlei",
+            "email": "kontakt@beberlei.de",
+            "role": "Project Founder"
+        },
+        {
+            "name": "Grégoire Pineau",
+            "email": "lyrixx@lyrixx.info",
+            "role": "Lead Developer"
```

Comment:
if you are the new lead, should the repository be kept on @beberlei's personal account or should it be moved to an organization to allow easier collaboration (and allowing @beberlei to send PRs) ?